cmd/cloudshell_open/main.go

```diff
@@ -308,6 +309,20 @@ func run(opts runOpts) error {
 		}
 	}
 
+	hookEnvs := append(envs,
+		fmt.Sprintf("PROJECT_ID=%s", project),
+		fmt.Sprintf("SERVICE=%s", serviceName),
+		fmt.Sprintf("REGION=%s", region))
+
+	if reflect.DeepEqual(existingService, service{}) {
```

Comment:
ideally we should not use `reflect` in main code (tests are ok).

Let's just return `*service` from describeService (or whatever) so we can do `service == nil` check.